docs/source/user-guide/vision/vision_properties.rst

```diff
@@ -32,6 +32,14 @@ Example for specific scenarios in which measuring properties may come in handy:
    the snow, the brightness of the image may be used to predict the label "wolf" easily. In this case, a model
    might not learn to discern wolf from dog by the animal's characteristics, but by using the background color.
 
+Passing Properties to the Checks
+================================
+The properties can be passed to the check in one of two ways:
+
+#. **Properties** - a list of functions to be calculated on the data during the check (specific format listed below)
```

Comment:
You can also select a subset of existing properties, no?

Reply:
@noamzbr I'm not aware of such option